### PR TITLE
New in Chrome 118 posts

### PR DESCRIPTION
**Pull request by tropicadri**
New in Chrome 118 posts

### CODE REVIEW

**Comment by tropicadri** [comment]
Thumbnail is not ready, but if you could review copy that would be great. Thanks!

File: site/en/blog/new-in-chrome-118/index.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+  - chrome-118
+---
+
+{% YouTube id='zx8OZtKe6hA' %}
+
+Here's what you need to know:
+
+* Declare specific styles within a component with the [`@scope` css rule](#css-scope).
+* There are two new media features: [`scripting` and `prefers-reduced-transparency`](#new-media-queries).
+* DevTools has improvements in the [**Sources** panel](#sources-panel-devtools)
```
Comment (rachelandrew) [inline site/en/blog/new-in-chrome-118/index.md:23]:
```suggestion
* DevTools has improvements in the [**Sources** panel](#sources-panel-devtools).
```

File: site/en/blog/new-in-chrome-118/index.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+without scope, the style applied is the last one declared.
+
+{% Compare 'worse', 'Without @scope' %}
+```css
+.pink-theme a { color: hotpink; }
+.lightpink-theme a { color: lightpink; }
+```
+{% endCompare %}
+
+{% Img src="image/SeARmcA1EicLXagFnVOe0ou9cqK2/N1ZbDIild8BHXJ3GjIey.jpg", alt="Two links, the first one reads &#39; I'm lightpink!' the second one reads 'Different pink', both links are actually light pink, under the links text reads sources order declaration style", width="800", height="700" %}
```
Comment (rachelandrew) [inline site/en/blog/new-in-chrome-118/index.md:52]:
```suggestion
{% Img src="image/SeARmcA1EicLXagFnVOe0ou9cqK2/N1ZbDIild8BHXJ3GjIey.jpg", alt="Two links, the first one reads &#39; I'm lightpink!' the second one reads 'Different pink', both links are actually light pink, under the links text reads sources order declaration style.", width="800", height="700" %}
```

File: site/en/blog/new-in-chrome-118/index.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+@scope (.lightpink-theme){
+    a {
+        color: lightpink;
+    }
+}
+```
+{% endCompare %}
+
+{% Img src="image/SeARmcA1EicLXagFnVOe0ou9cqK2/9svylwmK9QHQJUgAMNkj.jpg", alt="Two links, the first one reads ''I''m lightpink!'' the second one reads 'Different pink', second link is a darker pink, under the links text nearest ancestor style and has a green checkmark next to it", width="800", height="694" %}
```
Comment (rachelandrew) [inline site/en/blog/new-in-chrome-118/index.md:72]:
```suggestion
{% Img src="image/SeARmcA1EicLXagFnVOe0ou9cqK2/9svylwmK9QHQJUgAMNkj.jpg", alt="Two links, the first one reads ''I''m lightpink!'' the second one reads 'Different pink', second link is a darker pink, under the links text nearest ancestor style and has a green checkmark next to it.", width="800", height="694" %}
```

File: site/en/blog/new-in-chrome-118/index.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+}
+```
+{% endCompare %}
+
+
+With scope you can also style a component without styling certain things that are nested within. In a way you can have “holes” where the scoped style doesn't apply.
+
+Like in the following example, we could apply style to the text and exclude controls or vice versa.
+
+{% Img src="image/SeARmcA1EicLXagFnVOe0ou9cqK2/2OcZt7BEhhMH2dW9earg.jpg", alt="representation of the html above, with the words in scope next to the first and third div and the word excluded next to the second and fourth div", width="800", height="648" %}
```
Comment (rachelandrew) [inline site/en/blog/new-in-chrome-118/index.md:127]:
```suggestion
{% Img src="image/SeARmcA1EicLXagFnVOe0ou9cqK2/2OcZt7BEhhMH2dW9earg.jpg", alt="representation of the html above, with the words in scope next to the first and third div and the word excluded next to the second and fourth div.", width="800", height="648" %}
```

File: site/es/blog/new-in-chrome-118/index.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+  - chrome-118
+---
+
+{% YouTube id='zx8OZtKe6hA' %}
+
+Esto es lo que necesitas saber:
+
+* Declarar estilos específicos dentro de un componente con la [regla css `@scope`](#css-scope).
+* Hay dos nuevas funciones de medios: [`scripting` y `prefers-reduced-transparency`](#new-media-queries).
+* DevTools tiene mejoras en el [panel **Fuentes**](#sources-panel-devtools)
```
Comment (rachelandrew) [inline site/es/blog/new-in-chrome-118/index.md:23]:
```suggestion
* DevTools tiene mejoras en el [panel **Fuentes**](#sources-panel-devtools).
```

File: site/es/blog/new-in-chrome-118/index.md [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+sin`@scope`, el estilo aplicado es el último declarado.
+
+{% Compare 'worse', 'Sin @scope' %}
+```css
+.pink-theme a { color: hotpink; }
+.lightpink-theme a { color: lightpink; }
+```
+{% endCompare %}
+
+{% Img src="image/SeARmcA1EicLXagFnVOe0ou9cqK2/N1ZbDIild8BHXJ3GjIey.jpg", alt="Dos enlaces, el primero dice '¡Soy rosa claro!', el segundo dice 'Rosa diferente', ambos enlaces son en realidad rosa claro, debajo del el texto del enlace lee las fuentes estilo aplicadas según el orden de declaración (en inglés)", width="800", height="700" %}
```
Comment (rachelandrew) [inline site/es/blog/new-in-chrome-118/index.md:53]:
```suggestion
{% Img src="image/SeARmcA1EicLXagFnVOe0ou9cqK2/N1ZbDIild8BHXJ3GjIey.jpg", alt="Dos enlaces, el primero dice '¡Soy rosa claro!', el segundo dice 'Rosa diferente', ambos enlaces son en realidad rosa claro, debajo del el texto del enlace lee las fuentes estilo aplicadas según el orden de declaración (en inglés).", width="800", height="700" %}
```

File: site/es/blog/new-in-chrome-118/index.md [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+```
+{% endCompare %}
+
+
+
+Con `@scope` también puedes diseñar un componente sin aplicar el estilo a ciertas cosas que están anidadas en su interior. En cierto modo, puede tener "agujeros" donde no se aplica el estilo del scope.
+
+Como en el siguiente ejemplo, podríamos aplicar estilo al texto y excluir controles o viceversa.
+
+{% Img src="image/SeARmcA1EicLXagFnVOe0ou9cqK2/2OcZt7BEhhMH2dW9earg.jpg", alt="representación del html anterior, con las palabras dentro del scope junto al primer y tercer div y la palabra excluida junto al segundo y cuarto div", width="800", height="648" %}
```
Comment (rachelandrew) [inline site/es/blog/new-in-chrome-118/index.md:130]:
```suggestion
{% Img src="image/SeARmcA1EicLXagFnVOe0ou9cqK2/2OcZt7BEhhMH2dW9earg.jpg", alt="representación del html anterior, con las palabras dentro del scope junto al primer y tercer div y la palabra excluida junto al segundo y cuarto div.", width="800", height="648" %}
```

**Review by rachelandrew — APPROVED** [review]
Just a few missing bits of punctuation!

**Pull request merged**